features.bzl

```diff
@@ -33,6 +33,11 @@ _cc = struct(
     protobuf_on_allowlist = ge("8.0.0"),
 )
 
+_docs = struct(
+    # See https://bazelbuild.slack.com/archives/CDCMRLS23/p1727889648853179?thread_ts=1727887056.727699&cid=CDCMRLS23
```

Comment:
Slack links by themselves aren't great in committed code. Could you briefly describe why this is needed and also link the commit that introduced the change?

Reply:
I spent a few minutes searching the 300 or so commits since the last rolling release but couldn't find this one. @tetromino do you know offhand?

Reply:
@alexeagle - I think you're looking for https://github.com/bazelbuild/bazel/commit/bd1c3af2ea14e81268e940d2b8ba5ad00c3f08d7

Reply:
Thank you! Updated.

I see that https://github.com/bazelbuild/stardoc/pull/231/files is the renderer change in Stardoc that corresponds to this. I don't understand this fully - in my testing even the last rolling release of Bazel didn't print the `**` on kwargs even though the commit you point out has been included since [8.0.0-pre.20240603.2](https://github.com/bazelbuild/bazel/releases/tag/8.0.0-pre.20240603.2).

Reply:
It looks to me as if the `**` would only show up if you combine a Bazel with the linked commit and an up-to-date Stardoc.

Reply:
Well, in my testing I had the same version of Stardoc in both cases. But I'm pretty sure this PR accurately points to the Bazel version containing the change. Okay to merge this?